src/content/_index.en.md

```diff
@@ -19,7 +19,37 @@ Submariner is completely open source, and designed to be network plugin (CNI) ag
 * [subctl](./deployment/), a friendly deployment tool 
 * Support for interconnecting clusters with [overlapping CIDRs](./architecture/globalnet/)
 
+<style>
+.mygrid {  
+    display: grid;
+    grid-gap: 12px;  
+    grid-template-columns: repeat(auto-fit, minmax(180px, 1fr));
+    grid-template-rows: repeat(2, 100px);  
+    padding-bottom: 2em;
+}
+.mygrid h5 {
+    text-align: center;
+}
+
+</style>
+
+### Examples
```

Comment:
"Demos" seems a better title to me. I think we should also use ## and not ### here, to keep the title hierarchy of this page.

Reply:
:+1:

Reply:
```suggestion
### Demos
```